### PR TITLE
[Maintenance] Upgrade PHPUnit package

### PR DESCRIPTION
**Pull request by GSadee**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,6 +10,6 @@
 
 /tests/Application/yarn.lock
 
-.env.test
-
```
Comment (lchrusciel) [inline .gitignore:14]: Is this change needed? This file is not part of this PR

Comment (GSadee) [reply .gitignore:14]: Yup, but since I changed something in `.gitignore` file, I wanted to fix and delete a file that shouldn't be there as we have env files only in test application directory

**Comment by lchrusciel** [comment]
Thank you, Grzegorz! :tada:
